packages/core/src/client/decorators/bundlerClient.ts

```diff
@@ -58,8 +68,10 @@ export type BundlerActions = {
    * @params stateOverride - the state override to use for the estimation
    * @returns the gas estimates for the given response (see: {@link UserOperationEstimateGasResponse})
    */
-  estimateUserOperationGas(
-    request: UserOperationRequest,
+  estimateUserOperationGas<
+    TEntryPointVersion extends EntryPointVersion = DefaultEntryPointVersion
```

Comment:
```suggestion
    TEntryPointVersion extends EntryPointVersion = EntryPointVersion
```

Reply:
this one I think we talked about async, but this method should probs not default